person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(age, parent_permission, name = 'Unknown')
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+end
+
+private
+
+def of_age?
+  @age >= 18
+end
+
+public
+
+def can_use_services?
+  of_age? || @parent_permission
+end
```

Comment:
- [ ] Kindly remove **end** keyword from **line 8** and write **end** keyword after **line 20** because private and public sections should be inside the Person class 😊